### PR TITLE
Updated UI to Collection view 

### LINKED ISSUE #563: Updated UI to Collection view
I really like this updated UI for the Collection view:

image

The most important part is the "Add dataset to current collection" button:
image

This button here should say "Delete collection", but we have to figure out the logic of this option.

image

The thing is that every dataset has to belong to a collection. So if you delete the collection, what happens? It's not bad per se, but if you have a dataset that has no collection, then it should create a new collection when you try and reopen it. I'm not sure if that's currently the case.

### PR DESCRIPTION
**Pull request by bruyeret**
What do you think @arjunrajlab?

Fixes request changes of #598 
Close #563

### CODE REVIEW

**Comment by arjunrajlab** [comment]
Actually, never mind, I think there was an issue with that particular file, this works!

**Review by arjunrajlab — APPROVED** [review]
Looks great!

File: src/components/AddDatasetToCollection.vue [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+  set addDatasetOptionType(type: TAddDatasetOption["type"]) {
+    switch (type) {
+      case "add":
+        this.option = defaultDatasetAddOption();
+        break;
+      case "upload":
+        this.option = defaultDatasetUploadOption();
+        break;
+    }
```
Comment (luciemac) [inline src/components/AddDatasetToCollection.vue:180]: no default case ?

Comment (bruyeret) [reply src/components/AddDatasetToCollection.vue:180]: The two possible values for `type` are checked
TypeScript agrees with it and does not ask for more
I can add a `default:` above `case "upload":` but it should never be used

Comment (luciemac) [reply src/components/AddDatasetToCollection.vue:180]: sounds good, just wanted to make sure there is no possible troubles here

**Comment by luciemac** [comment]
LGTM

**Pull request merged**